9.2/run-postgresql.sh

```diff
@@ -16,27 +14,13 @@ else
   export PGDATA=$HOME/data/userdata
 fi
 
-if [ "$1" == "postgres-master" ]; then
-  shift
-  exec /usr/local/bin/run-postgresql-master.sh "$@"
-fi
-
-if [ "$1" == "postgres-slave" ]; then
-  shift
-  exec /usr/local/bin/run-postgresql-slave.sh "$@"
-fi
-
-generate_postgresql_config
-generate_passwd_file
-
-if [ "$1" = "postgres" ]; then
-  if [ ! -f "$PGDATA/postgresql.conf" ]; then
-    initialize_database
-  fi
-  pg_ctl -w start
-  set_passwords
-  pg_ctl stop
-fi
+case "${1:-}" in
+  'postgres-slave' )
```

Comment:
not so simple, see https://github.com/openshift/postgresql/pull/51#discussion-diff-38196215L24
It was already spelled that way :/